### PR TITLE
[HPRO-751] Replace Sensio security checker with Symfony CLI

### PR DESCRIPTION
**Pull request by jt2k**
| Q                   | A
| ------------------- | --------------
| Target Release?     | Next available
| Bug fix?            | ❌
| New feature?        | ✅
| Database migration? | ❌
| New configuration?  | ❌
| Composer updates?   | ✅
| NPM updates?        | ❌
| Jira Ticket(s)      | HPRO-751

### Summary

Our deploy script checks for known security vulnerabilities in Composer packages. It was using the `sensiolabs/security-checker` library, which is being removed. This PR replaces the same functionality with the Symfony CLI tool.

### Instructions for testing

1. Install the [Symfony CLI](https://symfony.com/download) tool if you don't already have it installed locally (README has been updated with this instruction)
2. Run `composer install` to remove `sensiolabs/security-checker`
3. Run `bin/console pmi:deploy --local` or another deploy command that would check for composer vulnerabilities.

To test a vulnerability, manually edit `composer.lock` to simulate a version of a package with a vulnerability. For example:

```diff
        {
            "name": "symfony/http-kernel",
-           "version": "v4.4.13",
+           "version": "v4.4.0",
```

### CODE REVIEW

File: src/Pmi/Console/Command/DeployCommand.php [hunk 1]
```diff
@@ -388,20 +386,13 @@ private function displayVulnerabilities($vulnerabilities)
...
-        } catch (HttpException $e) {
-            $this->out->writeln('<error>' . $e->getMessage() . '</error>');
-            if (!$helper->ask($this->in, $this->out, new ConfirmationQuestion('Continue anyways? '))) {
-                throw new \Exception('Aborting due to SensioLabs Security Checker network error');
-            }
-        }
-        // Ignore vulnerabilities mentioned in sensioignore file
-        $vulnerabilities = $this->removeSensioIgnoredVulnerabilities($vulnerabilities);
+        $this->out->writeln("Running Symfony Security Check...");
+        $process = new Process('symfony security:check --format=json --dir=' . escapeshellarg($this->appDir));
```
Comment (stephenyeargin) [inline src/Pmi/Console/Command/DeployCommand.php:390]: Only thing I might suggest here is running a `which symfony` check and throwing a console error if it evaluates false, prompting the developer to install it (or fix their `$PATH`).

**Review by stephenyeargin — APPROVED** [review]
:+1:

```
$ bin/console pmi:deploy --local
Checking URL handler security...
... all 8 handlers are secure.
Running Symfony Security Check...
No packages have known vulnerabilities

Running npm audit...

[...]
```

**Comment by jt2k** [comment]
Updated testing plan with instructions for simulating a vulnerability

**Review by stephenyeargin — APPROVED** [review]

**Review by shyamnaru — APPROVED** [review]

**Comment by jt2k** [comment]
@shyamnaru pointed out that the local deploy command is used when spinning up the Docker container. So I need to add the Symfony CLI to our Docker image. I originally thought I didn't need to since we deploy from our local machines, but I forgot that the local deploy is used by Docker. We could also consider skipping the security check when starting up the web server in Docker, since it doesn't provide much value running non-interactively.

**Comment by jt2k** [comment]
Added Symfony CLI to Docker image. You will need to rebuild the web image to test. The deploy command should now work from within Docker.

**Review by stephenyeargin — APPROVED** [review]

**Review by shyamnaru — APPROVED** [review]

**Pull request merged**